pkgs/applications/networking/mailreaders/astroid/default.nix

```diff
@@ -3,7 +3,7 @@
 , gtkmm3, libpeas, gsettings-desktop-schemas, gobject-introspection, python3
 
 # vim to be used, should support the GUI mode.
-, vim ? vim_configurable.override { features = "normal"; gui = "auto"; }
+, vim
```

Comment:
That needs to be converted into an override.

Reply:
I don't really want to fix all of the bugs that were already there in this PR. In this case, this `vim_configurable.override` was never even used, which this PR detected. So in fact, this doesn't change anything, and if it was broken before, it will still be broken (unless someone fixes it).

Reply:
I fixed it in https://github.com/NixOS/nixpkgs/pull/131787.